client/src/pages/LoginPage.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { login } from '../store/auth';
+import { Redirect } from 'react-router-dom';
+import { Container } from '@material-ui/core'
+import '../style/loginpage.css'
+
+function LoginPage() {
+    const [username, setUsername] = useState('')
+    const [password, setPassword] = useState('')
+    const currentUserId = useSelector(state => state.auth.id);
+    const dispatch = useDispatch();
+    
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        dispatch(login(username, password))
+    }
+
+    if(currentUserId) return <Redirect to="/profile" />;
+
+    return(
+        <>
```

Comment:
No need for the React Fragment since you have a parent `<div>` element wrapping all the inner elements.